releases/release-6.3.0.md

```diff
@@ -0,0 +1,303 @@
+
+* Improve the performance of adding indexes and reduce its impact on DML transactions [#35983](https://github.com/pingcap/tidb/issues/35983) @[benjamin2037](https://github.com/benjamin2037) **tw: Oreoxmt**
+
+    To improve the speed of backfilling when creating an index, TiDB v6.3.0 accelerates the `ADD INDEX` and `CREATE INDEX` DDL operations when the [`tidb_ddl_enable_fast_reorg`](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) system variable is enabled. The performance of adding indexes is about three times faster than the original when the feature is enabled.
+
+* Add support for `REGEXP_INSTR()`, `REGEXP_LIKE()`, `REGEXP_REPLACE()`, and `REGEXP_SUBSTR()` regular expression functions [#23881](https://github.com/pingcap/tidb/issues/23881) @[windtalker](https://github.com/windtalker) **tw: Oreoxmt**
+
+    For more details about the compatibility with MySQL, see [Regular expression compatibility with MySQL](/functions-and-operators/string-functions.md#regular-expression-compatibility-with-mysql).
+
+### Security
+
+* TiKV supports the SM4 algorithm for encryption at rest. [#13041](https://github.com/tikv/tikv/issues/13041) @[jiayang-zheng](https://github.com/jiayang-zheng)
+
+    Add the [SM4 algorithm](/encryption-at-rest.md) for TiKV encryption at rest. When you configure encryption at rest, you can enable the SM4 encryption capacity by setting the value of the "data-encryption-method" configuration to "sm4-ctr".
+
+* TiFlash supports the SM4 algorithm for encryption at rest. [#5714](https://github.com/pingcap/tiflash/issues/5714) @[lidezhu](https://github.com/lidezhu)
+
+    Add the [SM4 algorithm](/encryption-at-rest.md) for TiFlash encryption at rest. When you configure encryption at rest, you can enable the SM4 encryption capacity by setting the value of the "data-encryption-method" configuration to "sm4-ctr".
+
+* TiDB supports authentication with the SM3 algorithm [#36192](https://github.com/pingcap/tidb/issues/36192) @[CbcWestwolf](https://github.com/CbcWestwolf) **tw：ran-huang**
+
+    TiDB adds an authentication plugin [`tidb_sm3_password`](/system-variables.md#default_authentication_plugin) based on the SM3 algorithm. When this plugin is enabled, the user password is encrypted and validated using the SM3 algorithm.
+
+* JDBC supports authentication with the SM3 algorithm [issue]() @[lastincisor](https://github.com/lastincisor) **tw：ran-huang**
```

Comment:
```suggestion
```

I think the `tidb_sm3_password` authentication now uses the `caching_sha2_password` client side plugin. This makes it compatible with existing MySQL clients, connectors, etc. This probably means there is no JDBC specific support needed.

Reply:
@ljun0712 PTAL

Reply:
@dveeden We changed the compatibility of sm3 plugin in https://github.com/pingcap/tidb/pull/37909. Now TiDB JDBC is required to use sm3 plugin.  PTAL.

Reply:
I don't see any real issues with using `caching_sha2_password` as a clientside plugin. If that for some reason can't be used I think using [`mysql_clear_password`](https://dev.mysql.com/doc/refman/8.0/en/cleartext-pluggable-authentication.html) would have been better. Requiring a custom client side plugin that at least for now doesn't really do anything special is limiting compatibility a lot.

But that also means the release notes are correct, so this thread can be resolved.